src/test/packages/22-git-and-flux/zarf.yaml

```diff
@@ -38,20 +38,20 @@ components:
         before:
           # Check to verify the package secret has been saved for the already deployed component
           - cmd: test $(./zarf tools kubectl get secret -n zarf zarf-package-git-data-check-secrets -o jsonpath='{.data.*}' | base64 --decode | jq -r .deployedComponents | jq '. | length') -eq 2
-            description: "Check that the package secret has been updated with the deployed component"
+            description: Check that the package secret has been updated with the deployed component
             maxRetries: 3
 
   - name: specific-branch
     required: true
     repos:
       # Do a branch-provided Git Repo mirror
-      - "https://github.com/DoD-Platform-One/big-bang.git@refs/heads/release-1.54.x"
+      - https://github.com/DoD-Platform-One/big-bang.git@refs/heads/release-1.54.x
```

Comment:
One thing to consider here is doing a branch ref to zarf or something else we more directly control - we don't need repo1 for this and this can add an extra flake we don't need. (won't block the PR for it but could be worth doing)